backend/plugins/payment_api/src/widget/src/components/Phone.tsx

```
@@ -0,0 +1,182 @@
+import { createRef, memo, useEffect, useRef, useState } from 'react';
+import { Input } from './ui/input';
+
+const PhoneNumber = ({
+  inputRegExp = /^[0-9]$/,
+  amount = 8,
+  handleOutputString = () => {},
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)